deltacat/compute/compactor_v2/model/hash_bucket_input.py

```diff
@@ -22,6 +22,7 @@ def of(
         object_store: Optional[IObjectStore] = None,
         deltacat_storage=unimplemented_deltacat_storage,
         deltacat_storage_kwargs: Optional[Dict[str, Any]] = None,
+        memory_logs_enabled: bool = False,
```

Comment:
Do not default at multiple places. It makes changing defaults harder.

Reply:
What would the alternative be? I imagine it would have to be Optional, but that would essentially be the same behavior, right?

Reply:
Use like below:
```suggestion
        memory_logs_enabled: Optional[bool] = None,
```